plugins/workspace-backpack/src/backpack.js

```diff
@@ -497,6 +497,26 @@ export class Backpack extends Blockly.DragTarget {
     return JSON.stringify(json);
   }
 
+  /**
+   * Converts serialized XML to its equivalent serialized JSON string
+   * @param {!string} blockXml The XML serialized block.
```

Comment:
```suggestion
   * @param {string} blockXml The XML serialized block.
```

Primitives are non-null by default =)